subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -588,12 +588,7 @@ static int sm_send_bootstrap_registration(void)
 	LOG_DBG("Register ID with bootstrap server as '%s'",
 		log_strdup(query_buffer));
 
-	ret = lwm2m_send_message(msg);
-	if (ret < 0) {
-		LOG_ERR("Error sending LWM2M packet (err:%d).",
-			    ret);
-		goto cleanup;
-	}
+	lwm2m_send_message_async(msg);
```

Comment:
I just wonder now, given that `lwm2m_send_message_async()` cannot fail anyway, perhaps it should return `void` instead? Currently it might trigger some reports from Coverity, regarding unhandled return value.

Reply:
I really don't see anything that this function could return. Perhaps we could fix this with another separate change.

Reply:
Ok, we can sort it out in another PR.